code/modules/mob/language/languages.dm

```diff
@@ -228,3 +228,48 @@
 	color = "tajaran"
 	key = "7"
 	flags = RESTRICTED|HIVEMIND
+
+/datum/language/corticalborer
+	name = LANGUAGE_BORER
+	desc = "Cortical borers possess a strange link between their tiny minds."
+	speech_verb = "sings"
+	ask_verb = "sings"
+	exclaim_verb = "sings"
+	color = "alien"
+	key = "0"
```

Comment:
needs to be something different its messing up inetractions with other langauges